api/processors/auxil/log_event_creator.py

```diff
@@ -139,11 +139,13 @@ def create(
         coordinator: Coordinator,
         log_event_type: CoordinatorLogEventType,
         comment: str = "",
+        group: Group | None = None,
     ) -> None:
         created_log_event = CoordinatorLogEvent.objects.create(
             coordinator=coordinator,
             type=log_event_type,
             comment=comment,
+            group=group,
```

Comment:
Ideally we should check if the given event requires a `.group`, or, on the contrary, must not have any `.group` so that the user cannot create a wrong log event. What do you think?